### DISCUSSION TITLE
SSL on the Editor

### DISCUSSION THREAD

**Discussion by crisacostaoficial** [Budibase/budibase#5130] [Help]
HI,

I installed it on an ubuntu server.

I configured SSL in apache on port 80, but I didn't understand how to make the SSL certificate work in the Budibase editor.

In my case the port is 8080.

I used let's encryption

**Comment by melohagan** [comment]
Hi @crisacostaoficial 

I'm not too familiar with SSL, but I believe it is usually on port 443. 

Port 80 is for HTTP by default.

**Reply by crisacostaoficial** [reply]
Thanx! But, the Budibase was installed on the 8080.

Do u know?
